src/client/app/components/meters/EditMeterModalComponent.tsx

```diff
@@ -240,6 +243,22 @@ export default function EditMeterModalComponent(props: EditMeterModalComponentPr
 				props.meter.maxError != state.maxError ||
 				props.meter.disableChecks != state.disableChecks
 			);
+		let error_message = '';
+		for (const value of Object.values(groups)) {
+			for (let i = 0; i < value.deepMeters.length; i++) {
+				console.log(value.deepMeters[i], props.meter.id)
+				if(value.deepMeters[i]== props.meter.id)
+				{
+					error_message += translate('group') + value.name + ' uses ' + translate('meter') + ' "' + metersByID[value.deepMeters[i]].name + '" \n';
+				}
+			}
+		}
+		state.unitId = props.meter.unitId;
+
+		if (error_message) {
+			error_message = translate('meter.is.not.editable') + '\n' + error_message;
+			alert(error_message);
```

Comment:
I know alert is used elsewhere but we are trying to convert to Reaxct friendly alerts. Please look at src/client/app/utils/notifications.ts for success and error message functions. I think they would work here.

Also, I'm unclear that this stops the changes. After the alert it does not seem to have logic to stop the following code to update the meter.

Reply:
ok i made the change but now it displays both alerts. Im still figuring out how to fix that.